docs/03-architecture/05-security/05-keyreshare.md

```diff
@@ -0,0 +1,45 @@
+---
+slug: /readme/architecture/security/reshare
+id: security-reshare
+title: Key Resharing
+description: The following details the process for key resharing.
+sidebar_position: 5
+draft: false
+---
+
+:::info
+The following details the process for key resharing.
+:::
+
+Key resharing is an operation that makes multi-party threshold signing truly practical and secure under the realistic dynamic environment. 
+
+This operation allows for both:
+- Onboarding new relayers with new private key shares, and 
+- Recalibrating threshold requirements 
+
+This is done without the need to change the underlying private key, which is a delicate and expensive procedure.
+
+:::warning
+Our observations in the blockchain bridging space reveal that a big portion of vulnerabilities found are exploits of the key resharing mechanism. We presume that these vulnerabilities are as a result of sensitive key material as well as the operation needing to be repeated regularly.
```

Comment:
This is generally true, but IIRC vulns in key resharing can often be explained by the lack of well-defined protocols at their implementation. For ECDSA, the well-studied resharing proto was only introduced in the latest revision - [CGGMP21](https://eprint.iacr.org/2021/060).

Reply:
ive added the additional context!